src/cli/setup/create_node_vss.rs

```diff
@@ -0,0 +1,103 @@
+use crate::cli::setup::index_of;
+use crate::cli::setup::traits::Response;
+use crate::crypto::vss::{Commitment, Vss};
+use crate::errors::Error;
+use bitcoin::{PrivateKey, PublicKey};
+use clap::{App, Arg, ArgMatches, SubCommand};
+use std::collections::BTreeMap;
+use std::fmt;
+use std::str::FromStr;
+
+pub struct CreateNodeVssResponse {
+    vss: BTreeMap<PublicKey, Vss>,
+}
+
+impl CreateNodeVssResponse {
+    fn new(vss: BTreeMap<PublicKey, Vss>) -> Self {
+        CreateNodeVssResponse { vss: vss }
+    }
+}
+
+impl Response for CreateNodeVssResponse {}
+
+impl fmt::Display for CreateNodeVssResponse {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let c = self
+            .vss
+            .iter()
+            .map(|(k, v)| format!("{}:{}", k, v))
+            .collect::<Vec<String>>()
+            .join("\n");
+        write!(f, "{}", c)
+    }
+}
+
+pub struct CreateNodeVssCommand {}
+
+impl<'a> CreateNodeVssCommand {
+    pub fn execute(matches: &ArgMatches) -> Result<Box<dyn Response>, Error> {
+        let private_key: PrivateKey = matches
+            .value_of("private_key")
+            .and_then(|key| PrivateKey::from_wif(key).ok())
+            .ok_or(Error::InvalidArgs("private_key".to_string()))?;
+
+        let mut public_keys: Vec<PublicKey> = matches
+            .values_of("public_key")
+            .ok_or(Error::InvalidArgs("public_key".to_string()))?
+            .map(|key| PublicKey::from_str(key).map_err(|_| Error::InvalidKey))
+            .collect::<Result<Vec<PublicKey>, _>>()?;
+        public_keys.sort();
```

Comment:
The public keys sort should base on #74.

Reply:
This is applied to other PRs.(I'm going to omit this kind of comment to other PRs.)

Reply:
fixed at f84f9cd